build/hack/loadtest.go

```diff
@@ -0,0 +1,46 @@
+package hack
+
+import (
+	"context"
+
+	"dagger.io/dagger"
+	"github.com/google/uuid"
+)
+
+func LoadTest(ctx context.Context, client *dagger.Client, base, flipt *dagger.Container) error {
+	seed := base.File("build/testing/integration/readonly/testdata/default.yaml")
+	importCmd := []string{"/flipt", "import", "--create-namespace", "import.yaml"}
+
+	// import some test data
+	flipt, err := flipt.WithEnvVariable("UNIQUE", uuid.New().String()).
+		WithFile("import.yaml", seed).
+		WithExec(importCmd).
+		Sync(ctx)
+
+	if err != nil {
+		return err
+	}
+
+	flipt = flipt.WithEnvVariable("UNIQUE", uuid.New().String()).
+		WithExposedPort(8080).
+		WithExec(nil)
+
+	// build the loadtest binary
+	loadtest := base.
+		WithWorkdir("build/hack").
+		WithExec([]string{"go", "build", "-o", "./out/loadtest", "./cmd/loadtest/..."}).
+		File("out/loadtest")
+
+	// run the loadtest binary from within the pyroscope container and export the adhoc data
+	// output to the host
+	_, err = client.Container().
+		From("pyroscope/pyroscope:latest").
+		WithFile("loadtest", loadtest).
+		WithServiceBinding("flipt", flipt).
+		WithFile("loadtest", client.Host().Directory("build/hack/out").File("loadtest")).
```

Comment:
You can drop this line now.
```suggestion
```

Reply:
In fact if you dont I think it will error if thats not there. I assume it is there on your machine.